src/api/ratelimits.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Rate Limits
+sidebar_order: 5
+---
+
+Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each caller per endpoint.
```

Comment:
I don't understand the second sentence. If it makes sense, you could change it to "on a per endpoint basis" to make it grammatically correct, but I'm not sure. Feel free to ping me if you want to discuss to clarify this.
```suggestion
Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each caller per endpoint.
```